src/lib/iptables.ts

```diff
@@ -0,0 +1,30 @@
+import * as Promise from 'bluebird';
+import * as _childProcess from 'child_process';
+
+const childProcess: any = Promise.promisifyAll(_childProcess);
```

Comment:
It would be much better to do
```typescript
const execAsync = Promise.promisify(childProcess.exec)
```
as that's all you use and it will be able to create much better automatic typings than just using `any`